### PR TITLE
Huy/model

### PR DESCRIPTION
**Pull request by baohuy2k3**

### CODE REVIEW

File: app/Models/Option.php
Comment (ntd1683) [inline app/Models/Option.php:1]: Delete File

File: app/Models/Notify.php
Comment (ntd1683) [inline app/Models/Notify.php:1]: Delete File

File: app/Models/Banner.php
Comment (ntd1683) [inline app/Models/Banner.php:1]: Delete File

File: public/css/app.css
Comment (ntd1683) [inline public/css/app.css:1]: Delete File

File: app/Http/Controllers/UserController.php
Comment (ntd1683) [inline app/Http/Controllers/UserController.php:1]: create wrong file

File: app/Models/Address.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+class Address extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'address1',
+        'address2',
+        'district',
+        'province',
+        'user',
```
Comment (ntd1683) [inline app/Models/Address.php:17]:
```suggestion
        'use_id',
```

File: app/Models/Order.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Order extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'user',
```
Comment (ntd1683) [inline app/Models/Order.php:13]:
```suggestion
        'user_id',
```

File: app/Models/Order_Detail.php [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Order_Detail extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'order',
+        'product',
```
Comment (ntd1683) [inline app/Models/Order_Detail.php:14]:
```suggestion
        'product_id',
```

File: app/Models/Order_Detail.php [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Order_Detail extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'order',
```
Comment (ntd1683) [inline app/Models/Order_Detail.php:13]:
```suggestion
        'order_id',
```

File: app/Models/Payment_Detail.php [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Payment_Detail extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'order_code',
```
Comment (ntd1683) [inline app/Models/Payment_Detail.php:13]:
```suggestion
        'order_id',
```

File: app/Models/Product.php [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+use Illuminate\Database\Eloquent\Model;
+
+class Product extends Model
+{
+    use HasFactory;
+
+    protected $fillable =[
+        'name',
+        'SKU',
+        'category',
```
Comment (ntd1683) [inline app/Models/Product.php:15]:
```suggestion
        'product_category_id',
```

File: app/Models/Product_Detail.php [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Product_Detail extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'user',
```
Comment (ntd1683) [inline app/Models/Product_Detail.php:13]:
```suggestion
        'user_id',
```

File: app/Models/Product_Detail.php [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+        'product'
```
Comment (ntd1683) [inline app/Models/Product_Detail.php:14]:
```suggestion
        'product_id'
```

File: app/Models/Review_Ratings.php [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Review_Ratings extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'product',
+        'user',
```
Comment (ntd1683) [inline app/Models/Review_Ratings.php:14]:
```suggestion
        'product_id',
        'user_id',
```

File: app/Models/User_Payment.php [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class User_Payment extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'user',
```
Comment (ntd1683) [inline app/Models/User_Payment.php:13]:
```suggestion
        'user_id',
```

File: app/Models/Notify.php
Comment (baohuy2k3) [reply app/Models/Notify.php:1]: done

**Pull request closed without merging**